src/components/VaccineData/Map/index.js

```diff
@@ -0,0 +1,42 @@
+import * as React from "react";
+import mapboxgl from "!mapbox-gl"; // eslint-disable-line import/no-webpack-loader-syntax
+
+import "../SingleVaccineData/VaccineDataSingle.css";
+import { Button } from "@material-ui/core";
+import { CloseSharp } from "@material-ui/icons";
+mapboxgl.accessToken = process.env.REACT_APP_MAPBOX_ACCESS_TOKEN;
+
+export const Map = ({ lng: _lng, lat: _lat, close }) => {
+  const mapContainer = React.useRef(null);
+  const map = React.useRef(null);
+  const [lng, setLng] = React.useState(_lng);
+  const [lat, setLat] = React.useState(_lat);
+  const [zoom] = React.useState(10);
+
+  React.useEffect(() => {
```

Comment:
isn't importing useEffect and useState more reasonable in the first line itself?
we dont have to always do React.useEffect!

Please change these.